tools/namefixer.py

```diff
@@ -732,13 +732,17 @@
     "player->heldItemActionParam": "player->itemAction",
     "player->unk_A9C": "player->secretRumbleCharge",
     "player->unk_AA0": "player->closestSecretDistSq",
+    "player->unk_B2A": "player->getItemDrawIdPlusOne",
+    "player->getItemDrawId": "player->getItemDrawIdPlusOne",
```

Comment:
Is `"player->getItemDrawId": "player->getItemDrawIdPlusOne"` needed? Looks like it just applied to this branch?